setup.py

```diff
@@ -117,35 +103,32 @@
 # building, rather than unconditionally in the setup.py import or exec
 # We may make tripwire versions of build_ext, build_py, install
 need_cython = True
-try:
-    from nisext.sexts import get_comrec_build
-except ImportError: # No nibabel
-    msg = ('Need nisext package from nibabel installation'
-           ' - please install nibabel first')
-    pybuilder = derror_maker(du_build_py, msg)
-    extbuilder = derror_maker(du_build_ext, msg)
-else: # We have nibabel
-    pybuilder = get_comrec_build('dipy')
-    # Cython is a dependency for building extensions, iff we don't have stamped
-    # up pyx and c files.
-    build_ext, need_cython = cyproc_exts(EXTS,
-                                         info.CYTHON_MIN_VERSION,
-                                         'pyx-stamps')
-    # Add openmp flags if they work
-    simple_test_c = """int main(int argc, char** argv) { return(0); }"""
-    omp_test_c = """#include <omp.h>
-    int main(int argc, char** argv) { return(0); }"""
-    extbuilder = add_flag_checking(
-        build_ext, [[['/arch:SSE2'], [], simple_test_c, 'USING_VC_SSE2'],
-            [['-msse2', '-mfpmath=sse'], [], simple_test_c, 'USING_GCC_SSE2'],
-            [['-fopenmp'], ['-fopenmp'], omp_test_c, 'HAVE_OPENMP']], 'dipy')
+pybuilder = get_comrec_build('dipy')
+# Cython is a dependency for building extensions, iff we don't have stamped
+# up pyx and c files.
+build_ext, need_cython = cyproc_exts(EXTS,
+                                     info.CYTHON_MIN_VERSION,
+                                     'pyx-stamps')
+# Add openmp flags if they work
+simple_test_c = """int main(int argc, char** argv) { return(0); }"""
+omp_test_c = """#include <omp.h>
+int main(int argc, char** argv) { return(0); }"""
+extbuilder = add_flag_checking(
+    build_ext, [
+        [['/arch:SSE2'], [], simple_test_c, 'USING_VC_SSE2'],
```

Comment:
I'm afraid I didn't notice this one.  No, the flag checking logic hasn't changed, this is just an indentation difference.